### PR TITLE
Test unit circle

### PR DESCRIPTION
**Pull request by TeroFrondelius**
I tried my unit circle test and 852 of 100000 failed. This doesn't run so long, maybe this is worth adding to the repo. Or maybe other broken tests covers these issues?

### CODE REVIEW

**Comment by jw3126** [comment]
Do you know what the fail rate for this test ist with `Miniball.jl`? Adding one or a few explicit broken unitcirle examples would definitly be a good thing.
Having them as random tests is not as important, but you can add them if you like.

File: test/test_unit_circle.jl [hunk 1]
```diff
@@ -0,0 +1,24 @@
+using StaticArrays
+
+@testset "stastical unit circle test" begin
+    pts = Array{Array{Float64,1},1}(5)
```
Comment (jw3126) [inline test/test_unit_circle.jl]: Later you populate this array with `SVector`. So either change its eltype here or populate with plain vectors.

File: test/test_unit_circle.jl [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            x = cos(angle)
+            y = sin(angle)
+            pts[i] = @SVector [x, y]
+            if i == 1
+              pts[end] = @SVector [cos(angle + pi), sin(angle + pi)]
+            end
+        end
+
+        mb = miniball(pts)
+        origo = isapprox(mb[1],[0.0;0.0];atol=10*eps())
```
Comment (jw3126) [inline test/test_unit_circle.jl]: I find `10*eps()` very strict. Does this test still produce some failures if you do `sqrt(eps())`?

File: test/test_unit_circle.jl (hunk 1 above)
Comment (TeroFrondelius) [reply test/test_unit_circle.jl]: @jw3126 can you help to change the correct eltype. What should I type? My skills are more on the management than coding.

Comment (jw3126) [reply test/test_unit_circle.jl]: You could do `pts = Vector{SVector{2,Float64}}(5)`

File: test/test_unit_circle.jl (hunk 2 above)
Comment (TeroFrondelius) [reply test/test_unit_circle.jl]: @jw3126 after changing atol=sqrt(eps()) zero tests fails. I will push a commit with this change and close this pull request, if all test passes.

**Comment by TeroFrondelius** [comment]
100000/100000 tests passes after changing the reasonable tolerance. I will close this pull request, there is no need for these tests at the moment.

**Comment by jw3126** [comment]
Very cool that they pass now!

**Pull request closed without merging**